plugin/pkg/admission/eventratelimit/apis/eventratelimit/register.go

```diff
@@ -22,8 +22,11 @@ import (
 )
 
 var (
+	// SchemeBuilder is a pointer used to call AddToScheme
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
```

Comment:
I tried unexporting `SchemeBuilder` and tests would pass, but other packages in the tree are using the same scheme and some can't be unexported. Therefore I keep it here for consistency.